src/mip/coefficient_strengthening.jl

```diff
@@ -0,0 +1,176 @@
+"""
+Perform coefficient strengthening on each row and on each integer variables
+
+In particular, for the i-th constraints of the form
+aᵢ x ⩽ bᵢ and let xⱼ be a integer/binary variable
+If aᵢⱼ ⩾ d = bᵢ - Mᵢⱼ - aᵢⱼ (uⱼ - 1) > 0
+where Mᵢⱼ is maximal activity of the i-th row without considering xⱼ
+and uⱼ is upper bound of xⱼ, then
+aᵢⱼ ≔ aᵢⱼ - d
+bᵢ ≔ bᵢ - duⱼ
+Also perform coefficient strengthening on constraints of the form aᵢ x ⩾ cᵢ
+with similar update rule
+
+Skip ranged constraints, i.e, cᵢ ⩽ aᵢ x ⩽ bᵢ
+
+The new coefficients are stored by updating ps.pb0
+If some coefficient is reduced to 0, it is kept as 0
```

Comment:
```suggestion
If some coefficient is reduced to 0, it is explicitly stored as as a zero
in both ps.pb0.arows and ps.pb0.acols.
```
Not sure what you mean by "only the nonzeros are updated"